packages/bff/src/model/validators.ts

```diff
@@ -29,34 +15,6 @@ export const catalogProcessApiEServiceDescriptorCertifiedAttributesSatisfied = (
     toTenantWithOnlyAttributes(tenant)
   );
 
-export function isAgreementSubscribed(
```

Comment:
Why do you think these should be in `modelMappingUtils`?
I thought that here, in `validators`, we keep all asserts and checks that return a boolean while in `modelMappingUtils` we keep all the utils that transform data

Reply:
simply because this function is used to fill a property in API response, so I choose to put it in `modelMappingUtils.ts`

Reply:
I still think that validators would be a better place for `hasCertifiedAttributes`, `isAgreementSubscribed` and `isAgreementUpgradable`, since they could be used whenever we need to check these things; but I don't have a strong opinion in this, let's see what the others prefer too

Reply:
make sense , this [commit](https://github.com/pagopa/interop-be-monorepo/pull/836/commits/daf82b550ce4944fd12e801247e921c50ba9da39) shift those functions to `validators.ts`